install_yunohostv2

```diff
@@ -244,11 +237,15 @@ workaround_avahi_installation() {
         return 0
     fi
 
-    # Get a random unused uid between 500 and 999 (system-user)
-    local avahi_id=$((500 + RANDOM % 500))
+    # Get the first unused uid between 500 and 999 (system-user)
```

Comment:
Actually we **need** to be RANDOM to have chances to get rid of the bug.
Changing it like this will always return 500, so the bug is back (if I install 2 different containers, they will both use uid=500)